x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable_factory.ts

```diff
@@ -36,7 +45,7 @@ export class AnomalySwimlaneEmbeddableFactory
 
   public getDisplayName() {
     return i18n.translate('xpack.ml.components.jobAnomalyScoreEmbeddable.displayName', {
-      defaultMessage: 'ML anomaly swim lane',
+      defaultMessage: 'Anomaly swim lane',
```

Comment:
For the Anomaly swim lane embeddable, I suggest using a tooltip like this: "View anomaly detection results in a timeline."